lib/ts/stacks.ts

```diff
@@ -76,7 +76,66 @@ namespace Stacks {
 
         return Controller;
     }
+
     export function addController(name: string, controller: ControllerDefinition) {
         application.register(name, createController(controller));
-    };
+    }
+
+    export interface TooltipOptions {
+        placement: string;
+        classNames: string;
+    }
+
+    export function setTooltipHtml(element: Element, html: string, options?: TooltipOptions) {
```

Comment:
Having these wrapper functions is good, but let's have them wrap a "native" tooltip function call. I should be able to do all this from within the controller itself and not need outside JS support. See how we did it in the [modal controller](https://github.com/StackExchange/Stacks/pull/415/files#diff-6374cf76f80a96788d1735a9c3bc5efdR88)

Reply:
I can put it on the tooltip class but not use a native instance method because:

1. We may be dealing with an element that doesn't have a controller.
2. Even after we add the controller attribute, we won't have a controller until after Stimulus creates it.  If the element isn't yet in the DOM, it may never happen.